### PR TITLE
Added the trade section

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Added the trade section.
2) Added the bottom sheet for the trade creation.
3) Added the load of the trades during the startup proces..

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
Screenshot_1674062367

File: wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart [hunk 1]
```diff
@@ -202,7 +202,7 @@ class OwnerViewViewModel extends ChangeNotifier {
     );
 
     if (likesCountEither.isLeft()) {
-      LocaleKeys.something_wrong.tr().show();
+      // LocaleKeys.something_wrong.tr().show();
```
Comment (MikeSofaer) [inline wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart:205]: why not remove?

**Review by MikeSofaer — APPROVED** [review]

File: wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply wallet/lib/pages/detailed_asset_view/owner_view_view_model.dart:205]: Will enable it back once the big dipper start working again.

File: wallet/macos/Flutter/ephemeral/flutter_export_environment.sh [hunk 2]
```diff
@@ -3,12 +3,10 @@
 export "FLUTTER_ROOT=/Users/jawad/Downloads/flutter"
```
Comment (afti-githobo) [inline wallet/macos/Flutter/ephemeral/flutter_export_environment.sh:3]: i understand this has been an issue for a while and hasn't really been looked at, but these ephemeral files shouldn't really be in source control, should they?

**Review by afti-githobo — APPROVED** [review]
approving since removing ephemeral files and other build stuff from source control would be out-of-scope for this pr, but making a note that we should go ahead and do that

File: wallet/macos/Flutter/ephemeral/flutter_export_environment.sh (hunk 2 above)
Comment (kjawadDeveloper2) [reply wallet/macos/Flutter/ephemeral/flutter_export_environment.sh:3]: i am not sure about that.

**Pull request merged**